CuratedContent/TuringWay.md

```diff
@@ -0,0 +1,33 @@
+# The Turning Way: Reproducible and Ethical Science. 
+<!--deck text start-->
+An online handbook to reproducible, ethical, and collaborative computational and data science.
+<!--deck text end-->
+
+#### Contributed by [Jed Brown](https://github.com/jedbrown), [Mark C. Miller](https://github.com/markcmiller86 "Mark C. Miller GitHub Profile")
+#### Publication date: June 22, 2023
+
+Resource information | Details
+:--- | :--- 
+Article title  | [The Turing Way](https://the-turing-way.netlify.app/index.html), [DOI (most up to date)](http://doi.org/10.5281/zenodo.3332807)
+Authors | The Turing Way Community, Becky Arnold, Louise Bowler, Sarah Gibson, Patricia Herterich, Rosie Higman, … Kirstie Whitaker. (2019, March 25). The Turing Way: A Handbook for Reproducible Data Science (Version v0.0.4). Zenodo. 
```

Comment:
```suggestion
Authors | The Turing Way Community, (2022, Dec 21). The Turing Way: A handbook for reproducible, ethical and collaborative research. Zenodo. 
```
I think this from their GitHub readme is more future-proof way to cite. I'll see about updating the guidance on the website.
https://github.com/alan-turing-institute/the-turing-way#citing-the-turing-way